src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -211,6 +211,7 @@ void SwerveChassis::Drive(ChassisMovement &moveInfo)
     Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, m_networkTableName, string("Drive Option"), moveInfo.driveOption);
     Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, m_networkTableName, string("Heading Option"), moveInfo.headingOption);
 
+    LogInformation();
```

Comment:
@witcpalek  I think this was replace and updated in the other branch, we probably should remove it, what do you think?

Reply:
@tan560 I commented this out in the other branch, so when we update, it should go away